### DISCUSSION TITLE
Is there a way to input a script (processing layer) that runs on the LaTeX code before it is rendered to PDF?

### DISCUSSION THREAD

**Discussion by danielvartan** [quarto-dev/quarto-cli#7481] [Q&A] [answered]
### Description

Some limitations regarding rendering from Quarto to PDF could be addressed with an intermediary processing layer.

Example of application: The inclusion of `\legend` after figures and tables.

**Comment by mcanouil** [comment, marked as answer]
I don't know what exactly you want to achieve, but Quarto converts your document to markdown / Pandoc AST which is then handled by Pandoc itself.
You could use `format: latex` to have the latex code, then do whatever you want to compile the document yourself.

**Reply by danielvartan** [reply]
Oh, I see... I need to take some time to study Pandoc. Thanks for the prompt response @mcanouil!

**Reply by mcanouil** [reply]
Pandoc is emebedded inside Quarto.
I don't think you need to study Pandoc (although in the long run it could be useful).
As I said, you only need to use `format: latex` then you'll have your latex file for you to do manual edits and then compile using whatever LaTeX engine.
